### ISSUE TITLE
The Azure PowerShell session has not been properly initialized.

### ISSUE THREAD

**Issue by ADataGMan** [Azure/azure-powershell#11669]
## Description
I'm at my wit's end for trying to figure how to use PowerShell and Azure together. I've uninstalled every trace of PS7 I could find and did everything else I could think of to remove old modules and disable the PowerShell 2.0 engine in windows features. I've turned off my machine, rebooted it, and tried to remove any hint of PowerShell existing.

Then I installed Powershell 7 as of 4/20 and installed Az, MicrosoftPowerBIMgmt, and AzTable. Initially I thought this might be due to a bug in the integrated terminal of VSCode, but even in just the PowerShell 7 terminal window none of this works.

I get the error message "
$ctx = New-AzStorageContext -ConnectionString "DefaultEndpointsProtoc …
     |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The Azure PowerShell session has not been properly initialized.  Please import the module and try again.
" 
If I run  import-module az.storage then it doesn't give me an error message, but I get the same message I got above.

Maybe it's a connection thing? I try running the AzConnect command and get this experience: "
Connect-AzAccount
Connect-AzAccount: The 'Connect-AzAccount' command was found in the module 'Az.Accounts', but the module could not be loaded. For more information, run 'Import-Module Az.Accounts'.
Import-Module Az.Accounts
Import-Module: Assembly with same name is already loaded"

I have no other error messages to go on and no hints on what to do next. The only thing I haven't tried is completely wiping my hard drive and reinstalling Windows. 

## Steps to reproduce

Do a fresh install of PowerShell 7 and try to execute anything related to connecting or using storage on Azure. 

```
$ctx = New-AzStorageContext -ConnectionString "DefaultEndpointsProtocol=<table storage endpoint>"
Connect-AzAccount
Import-Module Az.Accounts
```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

## Module versions
```powershell

    Directory: C:\Users\Aaron\Documents\PowerShell\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Manifest   2.0.2                 AzTable                             Desk      {Add-AzTableRow, Get-AzTableRow, Get-AzTableRowAll, Get-AzTableRowByPartitionKeyRowKey…}
Binary     6.5.5                 Mdbc                                Desk

    Directory: C:\Program Files\PowerShell\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     3.7.0                 Az                                  Core,Desk
Script     1.7.5                 Az.Accounts                         Core,Desk {Disable-AzDataCollection, Disable-AzContextAutosave, Enable-AzDataCollection, Enable-AzContextAutosave…}
Script     1.1.1                 Az.Advisor                          Core,Desk {Get-AzAdvisorRecommendation, Enable-AzAdvisorRecommendation, Disable-AzAdvisorRecommendation, Get-AzAdvisorConfigur…
Script     1.0.3                 Az.Aks                              Core,Desk {Get-AzAks, New-AzAks, Remove-AzAks, Import-AzAksCredential…}
Script     1.1.2                 Az.AnalysisServices                 Core,Desk {Resume-AzAnalysisServicesServer, Suspend-AzAnalysisServicesServer, Get-AzAnalysisServicesServer, Remove-AzAnalysisS…
Script     1.4.0                 Az.ApiManagement                    Core,Desk {Add-AzApiManagementApiToProduct, Add-AzApiManagementProductToGroup, Add-AzApiManagementRegion, Add-AzApiManagementU…
Script     1.0.3                 Az.ApplicationInsights              Core,Desk {Get-AzApplicationInsights, New-AzApplicationInsights, Remove-AzApplicationInsights, Set-AzApplicationInsightsPricin…
Script     1.3.6                 Az.Automation                       Core,Desk {Get-AzAutomationHybridWorkerGroup, Remove-AzAutomationHybridWorkerGroup, Get-AzAutomationJobOutputRecord, Import-Az…
Script     2.0.2                 Az.Batch                            Core,Desk {Remove-AzBatchAccount, Get-AzBatchAccount, Get-AzBatchAccountKey, New-AzBatchAccount…}
Script     1.0.2                 Az.Billing                          Core,Desk {Get-AzBillingInvoice, Get-AzBillingPeriod, Get-AzEnrollmentAccount, Get-AzConsumptionBudget…}
Script     1.4.2                 Az.Cdn                              Core,Desk {Get-AzCdnProfile, Get-AzCdnProfileSsoUrl, New-AzCdnProfile, Remove-AzCdnProfile…}
Script     1.2.3                 Az.CognitiveServices                Core,Desk {Get-AzCognitiveServicesAccount, Get-AzCognitiveServicesAccountKey, Get-AzCognitiveServicesAccountSku, Get-AzCogniti…
Script     3.6.0                 Az.Compute                          Core,Desk {Remove-AzAvailabilitySet, Get-AzAvailabilitySet, New-AzAvailabilitySet, Update-AzAvailabilitySet…}
Script     1.0.3                 Az.ContainerInstance                Core,Desk {New-AzContainerGroup, Get-AzContainerGroup, Remove-AzContainerGroup, Get-AzContainerInstanceLog}
Script     1.1.1                 Az.ContainerRegistry                Core,Desk {New-AzContainerRegistry, Get-AzContainerRegistry, Update-AzContainerRegistry, Remove-AzContainerRegistry…}
Script     1.1.0                 Az.DataBoxEdge                      Core,Desk {Get-AzDataBoxEdgeJob, Get-AzDataBoxEdgeDevice, Invoke-AzDataBoxEdgeDevice, New-AzDataBoxEdgeDevice…}
Script     1.7.0                 Az.DataFactory                      Core,Desk {Set-AzDataFactoryV2, Update-AzDataFactoryV2, Get-AzDataFactoryV2, Remove-AzDataFactoryV2…}
Script     1.0.2                 Az.DataLakeAnalytics                Core,Desk {Get-AzDataLakeAnalyticsDataSource, New-AzDataLakeAnalyticsCatalogCredential, Remove-AzDataLakeAnalyticsCatalogCrede…
Script     1.2.8                 Az.DataLakeStore                    Core,Desk {Get-AzDataLakeStoreTrustedIdProvider, Remove-AzDataLakeStoreTrustedIdProvider, Remove-AzDataLakeStoreFirewallRule, …
Script     1.1.0                 Az.DeploymentManager                Core,Desk {Get-AzDeploymentManagerArtifactSource, New-AzDeploymentManagerArtifactSource, Set-AzDeploymentManagerArtifactSource…
Script     1.0.2                 Az.DevTestLabs                      Core,Desk {Get-AzDtlAllowedVMSizesPolicy, Get-AzDtlAutoShutdownPolicy, Get-AzDtlAutoStartPolicy, Get-AzDtlVMsPerLabPolicy…}
Script     1.1.2                 Az.Dns                              Core,Desk {Get-AzDnsRecordSet, New-AzDnsRecordConfig, Remove-AzDnsRecordSet, Set-AzDnsRecordSet…}
Script     1.2.3                 Az.EventGrid                        Core,Desk {New-AzEventGridTopic, Get-AzEventGridTopic, Set-AzEventGridTopic, New-AzEventGridTopicKey…}
Script     1.4.3                 Az.EventHub                         Core,Desk {New-AzEventHubNamespace, Get-AzEventHubNamespace, Set-AzEventHubNamespace, Remove-AzEventHubNamespace…}
Script     1.4.0                 Az.FrontDoor                        Core,Desk {New-AzFrontDoor, Get-AzFrontDoor, Set-AzFrontDoor, Remove-AzFrontDoor…}
Script     3.1.0                 Az.HDInsight                        Core,Desk {Get-AzHDInsightJob, New-AzHDInsightSqoopJobDefinition, Wait-AzHDInsightJob, New-AzHDInsightStreamingMapReduceJobDef…
Script     1.0.1                 Az.HealthcareApis                   Core,Desk {New-AzHealthcareApisService, Remove-AzHealthcareApisService, Set-AzHealthcareApisService, Get-AzHealthcareApisServi…
Script     2.3.0                 Az.IotHub                           Core,Desk {Add-AzIotHubKey, Get-AzIotHubEventHubConsumerGroup, Get-AzIotHubConnectionString, Get-AzIotHubJob…}
Script     1.5.2                 Az.KeyVault                         Core,Desk {Add-AzKeyVaultCertificate, Update-AzKeyVaultCertificate, Stop-AzKeyVaultCertificateOperation, Get-AzKeyVaultCertifi…
Script     1.3.2                 Az.LogicApp                         Core,Desk {Get-AzIntegrationAccountAgreement, Get-AzIntegrationAccountAssembly, Get-AzIntegrationAccountBatchConfiguration, Ge…
Script     1.1.3                 Az.MachineLearning                  Core,Desk {Move-AzMlCommitmentAssociation, Get-AzMlCommitmentAssociation, Get-AzMlCommitmentPlanUsageHistory, Remove-AzMlCommi…
Script     1.0.2                 Az.ManagedServices                  Core,Desk {Get-AzManagedServicesAssignment, New-AzManagedServicesAssignment, Remove-AzManagedServicesAssignment, Get-AzManaged…
Script     1.0.2                 Az.MarketplaceOrdering              Core,Desk {Get-AzMarketplaceTerms, Set-AzMarketplaceTerms}
Script     1.1.1                 Az.Media                            Core,Desk {Sync-AzMediaServiceStorageKey, Set-AzMediaServiceKey, Get-AzMediaServiceKey, Get-AzMediaServiceNameAvailability…}
Script     1.6.2                 Az.Monitor                          Core,Desk {Get-AzMetricDefinition, Get-AzMetric, Remove-AzLogProfile, Get-AzLogProfile…}
Script     2.4.0                 Az.Network                          Core,Desk {Add-AzApplicationGatewayAuthenticationCertificate, Get-AzApplicationGatewayAuthenticationCertificate, New-AzApplica…
Script     1.1.1                 Az.NotificationHubs                 Core,Desk {Get-AzNotificationHub, Get-AzNotificationHubAuthorizationRule, Get-AzNotificationHubListKey, Get-AzNotificationHubP…
Script     1.3.4                 Az.OperationalInsights              Core,Desk {New-AzOperationalInsightsAzureActivityLogDataSource, New-AzOperationalInsightsCustomLogDataSource, Disable-AzOperat…
Script     1.2.1                 Az.PolicyInsights                   Core,Desk {Get-AzPolicyEvent, Get-AzPolicyState, Get-AzPolicyStateSummary, Get-AzPolicyRemediation…}
Script     1.1.1                 Az.PowerBIEmbedded                  Core,Desk {Remove-AzPowerBIWorkspaceCollection, Get-AzPowerBIWorkspaceCollection, Get-AzPowerBIWorkspaceCollectionAccessKey, G…
Script     1.0.2                 Az.PrivateDns                       Core,Desk {Get-AzPrivateDnsZone, Remove-AzPrivateDnsZone, Set-AzPrivateDnsZone, New-AzPrivateDnsZone…}
Script     2.8.0                 Az.RecoveryServices                 Core,Desk {Get-AzRecoveryServicesBackupProperty, Get-AzRecoveryServicesVault, Get-AzRecoveryServicesVaultSettingsFile, New-AzR…
Script     1.2.1                 Az.RedisCache                       Core,Desk {Remove-AzRedisCachePatchSchedule, New-AzRedisCacheScheduleEntry, Get-AzRedisCachePatchSchedule, New-AzRedisCachePat…
Script     1.0.3                 Az.Relay                            Core,Desk {New-AzRelayNamespace, Get-AzRelayNamespace, Set-AzRelayNamespace, Remove-AzRelayNamespace…}
Script     1.13.0                Az.Resources                        Core,Desk {Get-AzProviderOperation, Remove-AzRoleAssignment, Get-AzRoleAssignment, New-AzRoleAssignment…}
Script     1.4.1                 Az.ServiceBus                       Core,Desk {New-AzServiceBusNamespace, Get-AzServiceBusNamespace, Set-AzServiceBusNamespace, Remove-AzServiceBusNamespace…}
Script     2.0.1                 Az.ServiceFabric                    Core,Desk {Add-AzServiceFabricClientCertificate, Add-AzServiceFabricClusterCertificate, Add-AzServiceFabricNode, Add-AzService…
Script     1.1.1                 Az.SignalR                          Core,Desk {New-AzSignalR, Get-AzSignalR, Get-AzSignalRKey, New-AzSignalRKey…}
Script     2.5.0                 Az.Sql                              Core,Desk {Get-AzSqlDatabaseTransparentDataEncryption, Get-AzSqlDatabaseTransparentDataEncryptionActivity, Set-AzSqlDatabaseTr…
Script     1.1.0                 Az.SqlVirtualMachine                Core,Desk {New-AzSqlVM, Get-AzSqlVM, Update-AzSqlVM, Remove-AzSqlVM…}
Script     1.13.0                Az.Storage                          Core,Desk {Get-AzStorageAccount, Get-AzStorageAccountKey, New-AzStorageAccount, New-AzStorageAccountKey…}
Script     1.2.3                 Az.StorageSync                      Core,Desk {Invoke-AzStorageSyncCompatibilityCheck, New-AzStorageSyncService, Get-AzStorageSyncService, Remove-AzStorageSyncSer…
Script     1.0.1                 Az.StreamAnalytics                  Core,Desk {Get-AzStreamAnalyticsFunction, Get-AzStreamAnalyticsDefaultFunctionDefinition, New-AzStreamAnalyticsFunction, Remov…
Script     1.0.0                 Az.Support                          Core,Desk {Get-AzSupportService, Get-AzSupportProblemClassification, Get-AzSupportTicket, Get-AzSupportTicketCommunication…}
Script     1.0.3                 Az.TrafficManager                   Core,Desk {Add-AzTrafficManagerCustomHeaderToEndpoint, Remove-AzTrafficManagerCustomHeaderFromEndpoint, Add-AzTrafficManagerCu…
Script     1.8.0                 Az.Websites                         Core,Desk {Get-AzAppServicePlan, Set-AzAppServicePlan, New-AzAppServicePlan, Remove-AzAppServicePlan…}
Manifest   2.0.2                 AzTable                             Desk      {Add-AzTableRow, Get-AzTableRow, Get-AzTableRowAll, Get-AzTableRowByPartitionKeyRowKey…}
Manifest   1.0.840               MicrosoftPowerBIMgmt                Desk
Binary     1.0.840               MicrosoftPowerBIMgmt.Admin          Desk      {Add-PowerBIEncryptionKey, Get-PowerBIEncryptionKey, Get-PowerBIWorkspaceEncryptionStatus, Switch-PowerBIEncryptionK…
Binary     1.0.840               MicrosoftPowerBIMgmt.Capacities     Desk      Get-PowerBICapacity
Binary     1.0.840               MicrosoftPowerBIMgmt.Data           Desk      {Add-PowerBIDataset, Set-PowerBITable, New-PowerBIDataset, New-PowerBITable…}
Binary     1.0.840               MicrosoftPowerBIMgmt.Profile        Desk      {Connect-PowerBIServiceAccount, Disconnect-PowerBIServiceAccount, Invoke-PowerBIRestMethod, Get-PowerBIAccessToken…}
Binary     1.0.840               MicrosoftPowerBIMgmt.Reports        Desk      {Get-PowerBIReport, New-PowerBIReport, Export-PowerBIReport, Get-PowerBIDashboard…}
Binary     1.0.840               MicrosoftPowerBIMgmt.Workspaces     Desk      {Get-PowerBIWorkspace, Get-PowerBIWorkspaceMigrationStatus, Add-PowerBIWorkspaceUser, Remove-PowerBIWorkspaceUser…}

    Directory: C:\program files\powershell\7\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Manifest   7.0.0.0               CimCmdlets                          Core      {Get-CimAssociatedInstance, Get-CimClass, Get-CimInstance, Get-CimSession…}
Manifest   1.2.5                 Microsoft.PowerShell.Archive        Desk      {Compress-Archive, Expand-Archive}
Manifest   7.0.0.0               Microsoft.PowerShell.Diagnostics    Core      {Get-WinEvent, New-WinEvent, Get-Counter}
Manifest   7.0.0.0               Microsoft.PowerShell.Host           Core      {Start-Transcript, Stop-Transcript}
Manifest   7.0.0.0               Microsoft.PowerShell.Management     Core      {Add-Content, Clear-Content, Get-Clipboard, Set-Clipboard…}
Manifest   7.0.0.0               Microsoft.PowerShell.Security       Core      {Get-Acl, Set-Acl, Get-PfxCertificate, Get-Credential…}
Manifest   7.0.0.0               Microsoft.PowerShell.Utility        Core      {Export-Alias, Get-Alias, Import-Alias, New-Alias…}
Manifest   7.0.0.0               Microsoft.WSMan.Management          Core      {Disable-WSManCredSSP, Enable-WSManCredSSP, Get-WSManCredSSP, Set-WSManQuickConfig…}
Script     1.4.6                 PackageManagement                   Desk      {Find-Package, Get-Package, Get-PackageProvider, Get-PackageSource…}
Script     2.2.3                 PowerShellGet                       Desk      {Find-Command, Find-DSCResource, Find-Module, Find-RoleCapability…}
Script     2.0.5                 PSDesiredStateConfiguration         Core      {Configuration, New-DscChecksum, Get-DscResource, Invoke-DscResource}
Script     7.0.0.0               PSDiagnostics                       Core      {Disable-PSTrace, Disable-PSWSManCombinedTrace, Disable-WSManTrace, Enable-PSTrace…}
Script     2.0.0                 PSReadLine                          Desk      {Get-PSReadLineKeyHandler, Set-PSReadLineKeyHandler, Remove-PSReadLineKeyHandler, Get-PSReadLineOption…}
Binary     2.0.3                 ThreadJob                           Desk      Start-ThreadJob

    Directory: C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Manifest   1.0.0.0               AppBackgroundTask                   Core,Desk {Disable-AppBackgroundTaskDiagnosticLog, Enable-AppBackgroundTaskDiagnosticLog, Set-AppBackgroundTaskResourcePolicy,…
Manifest   2.0.1.0               Appx                                Core,Desk {Add-AppxPackage, Get-AppxPackage, Get-AppxPackageManifest, Remove-AppxPackage…}
Script     1.0.0.0               AssignedAccess                      Core,Desk {Clear-AssignedAccess, Get-AssignedAccess, Set-AssignedAccess}
Manifest   1.0.0.0               BitLocker                           Core,Desk {Unlock-BitLocker, Suspend-BitLocker, Resume-BitLocker, Remove-BitLockerKeyProtector…}
Manifest   1.0.0.0               BranchCache                         Core,Desk {Add-BCDataCacheExtension, Clear-BCCache, Disable-BC, Disable-BCDowngrading…}
Manifest   1.0                   ConfigDefender                      Core,Desk {Get-MpPreference, Set-MpPreference, Add-MpPreference, Remove-MpPreference…}
Manifest   1.0                   Defender                            Core,Desk {Get-MpPreference, Set-MpPreference, Add-MpPreference, Remove-MpPreference…}
Manifest   1.0.2.0               DeliveryOptimization                Core,Desk {Delete-DeliveryOptimizationCache, Get-DeliveryOptimizationStatus, Set-DeliveryOptimizationStatus, Get-DeliveryOptim…
Manifest   1.0.0.0               DirectAccessClientComponents        Core,Desk {Disable-DAManualEntryPointSelection, Enable-DAManualEntryPointSelection, Get-DAClientExperienceConfiguration, Get-D…
Script     3.0                   Dism                                Core,Desk {Add-AppxProvisionedPackage, Add-WindowsDriver, Add-WindowsCapability, Add-WindowsImage…}
Manifest   1.0.0.0               DnsClient                           Core,Desk {Resolve-DnsName, Clear-DnsClientCache, Get-DnsClient, Get-DnsClientCache…}
Manifest   1.0.0.0               EventTracingManagement              Core,Desk {Start-EtwTraceSession, New-EtwTraceSession, Get-EtwTraceSession, Update-EtwTraceSession…}
Manifest   1.0.0.0               HgsClient                           Core,Desk {Get-HgsAttestationBaselinePolicy, Get-HgsClientConfiguration, Test-HgsClientConfiguration, Set-HgsClientConfigurati…
Manifest   1.0.0.0               HgsDiagnostics                      Core,Desk {New-HgsTraceTarget, Get-HgsTrace, Get-HgsTraceFileData, Test-HgsTraceTarget}
Binary     2.0.0.0               Hyper-V                             Core,Desk {Add-VMAssignableDevice, Add-VMDvdDrive, Add-VMFibreChannelHba, Add-VMGpuPartitionAdapter…}
Binary     1.1                   Hyper-V                             Core,Desk {Add-VMDvdDrive, Add-VMFibreChannelHba, Add-VMHardDiskDrive, Add-VMMigrationNetwork…}
Manifest   2.0.0.0               International                       Core,Desk {Get-WinDefaultInputMethodOverride, Set-WinDefaultInputMethodOverride, Get-WinHomeLocation, Set-WinHomeLocation…}
Manifest   3.0.0.0               Microsoft.PowerShell.Diagnostics    Core,Desk {Get-WinEvent, Get-Counter, Import-Counter, Export-Counter…}
Manifest   1.0.0.0               Microsoft.PowerShell.LocalAccounts  Core,Desk {Add-LocalGroupMember, Disable-LocalUser, Enable-LocalUser, Get-LocalGroup…}
Manifest   3.1.0.0               Microsoft.PowerShell.Management     Core,Desk {Add-Content, Clear-Content, Clear-ItemProperty, Join-Path…}
Manifest   1.0                   MMAgent                             Core,Desk {Disable-MMAgent, Enable-MMAgent, Set-MMAgent, Get-MMAgent…}
Manifest   2.0.0.0               NetAdapter                          Core,Desk {Disable-NetAdapter, Disable-NetAdapterBinding, Disable-NetAdapterChecksumOffload, Disable-NetAdapterEncapsulatedPac…
Manifest   1.0.0.0               NetConnection                       Core,Desk {Get-NetConnectionProfile, Set-NetConnectionProfile}
Manifest   1.0.0.0               NetEventPacketCapture               Core,Desk {New-NetEventSession, Remove-NetEventSession, Get-NetEventSession, Set-NetEventSession…}
Manifest   2.0.0.0               NetLbfo                             Core,Desk {Add-NetLbfoTeamMember, Add-NetLbfoTeamNic, Get-NetLbfoTeam, Get-NetLbfoTeamMember…}
Manifest   1.0.0.0               NetNat                              Core,Desk {Get-NetNat, Get-NetNatExternalAddress, Get-NetNatStaticMapping, Get-NetNatSession…}
Manifest   2.0.0.0               NetQos                              Core,Desk {Get-NetQosPolicy, Set-NetQosPolicy, Remove-NetQosPolicy, New-NetQosPolicy}
Manifest   2.0.0.0               NetSecurity                         Core,Desk {Get-DAPolicyChange, New-NetIPsecAuthProposal, New-NetIPsecMainModeCryptoProposal, New-NetIPsecQuickModeCryptoPropos…
Manifest   1.0.0.0               NetSwitchTeam                       Core,Desk {New-NetSwitchTeam, Remove-NetSwitchTeam, Get-NetSwitchTeam, Rename-NetSwitchTeam…}
Manifest   1.0.0.0               NetTCPIP                            Core,Desk {Get-NetIPAddress, Get-NetIPInterface, Get-NetIPv4Protocol, Get-NetIPv6Protocol…}
Manifest   1.0.0.0               NetworkConnectivityStatus           Core,Desk {Get-DAConnectionStatus, Get-NCSIPolicyConfiguration, Reset-NCSIPolicyConfiguration, Set-NCSIPolicyConfiguration}
Manifest   1.0.0.0               NetworkSwitchManager                Core,Desk {Disable-NetworkSwitchEthernetPort, Enable-NetworkSwitchEthernetPort, Get-NetworkSwitchEthernetPort, Remove-NetworkS…
Manifest   1.0.0.0               NetworkTransition                   Core,Desk {Add-NetIPHttpsCertBinding, Disable-NetDnsTransitionConfiguration, Disable-NetIPHttpsProfile, Disable-NetNatTransiti…
Manifest   1.0.0.0               PcsvDevice                          Core,Desk {Get-PcsvDevice, Start-PcsvDevice, Stop-PcsvDevice, Restart-PcsvDevice…}
Manifest   1.0.0.0               PnpDevice                           Core,Desk {Get-PnpDevice, Get-PnpDeviceProperty, Enable-PnpDevice, Disable-PnpDevice}
Manifest   1.1                   PrintManagement                     Core,Desk {Add-Printer, Add-PrinterDriver, Add-PrinterPort, Get-PrintConfiguration…}
Binary     1.0.11                ProcessMitigations                  Core,Desk {Get-ProcessMitigation, Set-ProcessMitigation, ConvertTo-ProcessMitigationPolicy}
Manifest   1.0.0.0               ScheduledTasks                      Core,Desk {Get-ScheduledTask, Set-ScheduledTask, Register-ScheduledTask, Unregister-ScheduledTask…}
Manifest   2.0.0.0               SecureBoot                          Core,Desk {Confirm-SecureBootUEFI, Set-SecureBootUEFI, Get-SecureBootUEFI, Format-SecureBootUEFI…}
Manifest   2.0.0.0               SmbShare                            Core,Desk {Get-SmbShare, Remove-SmbShare, Set-SmbShare, Block-SmbShareAccess…}
Manifest   2.0.0.0               SmbWitness                          Core,Desk {Get-SmbWitnessClient, Move-SmbWitnessClient, gsmbw, msmbw…}
Manifest   1.0.0.0               StartLayout                         Core,Desk {Export-StartLayout, Import-StartLayout, Export-StartLayoutEdgeAssets, Get-StartApps}
Manifest   2.0.0.0               Storage                             Core,Desk {Add-InitiatorIdToMaskingSet, Add-PartitionAccessPath, Add-PhysicalDisk, Add-StorageFaultDomain…}
Manifest   1.0.0.0               TroubleshootingPack                 Core,Desk {Get-TroubleshootingPack, Invoke-TroubleshootingPack}
Manifest   2.0.0.0               TrustedPlatformModule               Core,Desk {Get-Tpm, Initialize-Tpm, Clear-Tpm, Unblock-Tpm…}
Binary     2.1.639.0             UEV                                 Core,Desk
Manifest   2.0.0.0               VpnClient                           Core,Desk {Add-VpnConnection, Set-VpnConnection, Remove-VpnConnection, Get-VpnConnection…}
Manifest   1.0.0.0               Wdac                                Core,Desk {Get-OdbcDriver, Set-OdbcDriver, Get-OdbcDsn, Add-OdbcDsn…}
Manifest   2.0.0.0               Whea                                Core,Desk {Get-WheaMemoryPolicy, Set-WheaMemoryPolicy}
Manifest   1.0.0.0               WindowsDeveloperLicense             Core,Desk {Get-WindowsDeveloperLicense, Unregister-WindowsDeveloperLicense, Show-WindowsDeveloperLicenseRegistration}
Script     1.0                   WindowsErrorReporting               Core,Desk {Enable-WindowsErrorReporting, Disable-WindowsErrorReporting, Get-WindowsErrorReporting}
Manifest   1.0.0.0               WindowsSearch                       Core,Desk {Get-WindowsSearchSetting, Set-WindowsSearchSetting}
Manifest   1.0.0.0               WindowsUpdate                       Core,Desk Get-WindowsUpdateLog
Manifest   1.0.0.2               WindowsUpdateProvider               Core,Desk {Get-WUAVersion, Get-WULastInstallationDate, Get-WULastScanSuccessDate, Get-WUIsPendingReboot…}

    Directory: C:\Program Files\Intel\Wired Networking

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Binary     22.6.6.0              IntelNetCmdlets                     Desk

    Directory: C:\Program Files (x86)\Microsoft SQL Server\150\Tools\PowerShell\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Manifest   15.0                  SQLPS                               Desk
```

## Debug output

```
PS F:\Work\XCM> Connect-AzAccount
Connect-AzAccount: The 'Connect-AzAccount' command was found in the module 'Az.Accounts', but the module could not be loaded. For more information, run 'Import-Module Az.Accounts'.
PS F:\Work\XCM> Import-Module Az.Accounts
Import-Module: Assembly with same name is already loaded
```

## Error output
```
Resolve-AzError: The 'Resolve-AzError' command was found in the module 'Az.Accounts', but the module could not be loaded. For more information, run 'Import-Module Az.Accounts'.
```

**Comment by dingmeng-xue** [comment]
@ADataGMan , could you try below code and check if there is any problem?
```
Import-Module -name Az.Storage -debug
Import-Module -name Az.Accounts -debug
```

**Comment by ADataGMan** [comment]
I didn't see any errors, this is the result of what I received when I ran it. 

Afterwards I tried running Connect-AzAccount and it seemed to work fine. Maybe I just needed a few hard shutdowns, updates, and restarts to get it to download? 

Az.Storage results in:
```
PS C:\Users\Aaron> Import-Module -name Az.Storage -debug
VERBOSE: Loading module from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Az.Storage.psd1'.

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"): a
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Management.Storage.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Management.Storage.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.Common.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.Common.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.Blob.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.Blob.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.File.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.File.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.Queue.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.Queue.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Cosmos.Table.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Cosmos.Table.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.DocumentDB.Core.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.DocumentDB.Core.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.DataMovement.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.Storage.DataMovement.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.DocumentDB.Core.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.DocumentDB.Core.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.OData.Core.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.OData.Core.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.OData.Edm.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.OData.Edm.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Spatial.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Spatial.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.KeyVault.Core.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Microsoft.Azure.KeyVault.Core.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Azure.Storage.Blobs.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Azure.Storage.Blobs.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Azure.Storage.Common.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Azure.Storage.Common.dll'.
VERBOSE: Loading 'Assembly' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Azure.Storage.Files.DataLake.dll'.
VERBOSE: Loading 'Executable' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Azure.Storage.Files.DataLake.dll'.
VERBOSE: Loading 'FormatsToProcess' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Storage.Management.format.ps1xml'.
VERBOSE: Loading 'FormatsToProcess' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Storage.format.ps1xml'.
VERBOSE: Loading 'FormatsToProcess' from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Storage.generated.format.ps1xml'.
VERBOSE: Populating RepositorySourceLocation property for module Az.Storage.
VERBOSE: Loading module from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Storage\1.14.0\Az.Storage.psm1'.
VERBOSE: Exporting function 'Test-DotNet'.
VERBOSE: Exporting cmdlet 'Add-AzRmStorageContainerLegalHold'.
VERBOSE: Exporting cmdlet 'Disable-AzStorageBlobDeleteRetentionPolicy'.
VERBOSE: Exporting cmdlet 'Enable-AzStorageBlobDeleteRetentionPolicy'.
VERBOSE: Exporting cmdlet 'Get-AzStorageBlobServiceProperty'.
VERBOSE: Exporting cmdlet 'Get-AzRmStorageContainer'.
VERBOSE: Exporting cmdlet 'Get-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Exporting cmdlet 'Lock-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Exporting cmdlet 'New-AzRmStorageContainer'.
VERBOSE: Exporting cmdlet 'Remove-AzRmStorageContainer'.
VERBOSE: Exporting cmdlet 'Remove-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Exporting cmdlet 'Remove-AzRmStorageContainerLegalHold'.
VERBOSE: Exporting cmdlet 'Set-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Exporting cmdlet 'Update-AzStorageBlobServiceProperty'.
VERBOSE: Exporting cmdlet 'Update-AzRmStorageContainer'.
VERBOSE: Exporting cmdlet 'Get-AzRmStorageShare'.
VERBOSE: Exporting cmdlet 'New-AzRmStorageShare'.
VERBOSE: Exporting cmdlet 'Remove-AzRmStorageShare'.
VERBOSE: Exporting cmdlet 'Update-AzRmStorageShare'.
VERBOSE: Exporting cmdlet 'Add-AzStorageAccountManagementPolicyAction'.
VERBOSE: Exporting cmdlet 'Add-AzStorageAccountNetworkRule'.
VERBOSE: Exporting cmdlet 'Get-AzStorageAccount'.
VERBOSE: Exporting cmdlet 'Get-AzStorageAccountKey'.
VERBOSE: Exporting cmdlet 'Get-AzStorageAccountManagementPolicy'.
VERBOSE: Exporting cmdlet 'Get-AzStorageAccountNetworkRuleSet'.
VERBOSE: Exporting cmdlet 'New-AzStorageAccount'.
VERBOSE: Exporting cmdlet 'New-AzStorageAccountKey'.
VERBOSE: Exporting cmdlet 'New-AzStorageAccountManagementPolicyFilter'.
VERBOSE: Exporting cmdlet 'New-AzStorageAccountManagementPolicyRule'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageAccount'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageAccountManagementPolicy'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageAccountNetworkRule'.
VERBOSE: Exporting cmdlet 'Revoke-AzStorageAccountUserDelegationKeys'.
VERBOSE: Exporting cmdlet 'Set-AzCurrentStorageAccount'.
VERBOSE: Exporting cmdlet 'Set-AzStorageAccount'.
VERBOSE: Exporting cmdlet 'Set-AzStorageAccountManagementPolicy'.
VERBOSE: Exporting cmdlet 'Update-AzStorageAccountNetworkRuleSet'.
VERBOSE: Exporting cmdlet 'Get-AzStorageAccountNameAvailability'.
VERBOSE: Exporting cmdlet 'Get-AzStorageUsage'.
VERBOSE: Exporting cmdlet 'Set-AzDataLakeGen2ItemAclObject'.
VERBOSE: Exporting cmdlet 'Get-AzStorageTableStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Get-AzStorageTable'.
VERBOSE: Exporting cmdlet 'New-AzStorageTableSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageTableStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'New-AzStorageTable'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageTableStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageTable'.
VERBOSE: Exporting cmdlet 'Set-AzStorageTableStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Get-AzStorageQueue'.
VERBOSE: Exporting cmdlet 'New-AzStorageQueue'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageQueue'.
VERBOSE: Exporting cmdlet 'Get-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'New-AzStorageQueueSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Set-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Close-AzStorageFileHandle'.
VERBOSE: Exporting cmdlet 'Get-AzStorageFile'.
VERBOSE: Exporting cmdlet 'Get-AzStorageFileContent'.
VERBOSE: Exporting cmdlet 'Get-AzStorageFileCopyState'.
VERBOSE: Exporting cmdlet 'Get-AzStorageFileHandle'.
VERBOSE: Exporting cmdlet 'Get-AzStorageShare'.
VERBOSE: Exporting cmdlet 'Get-AzStorageShareStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'New-AzStorageDirectory'.
VERBOSE: Exporting cmdlet 'New-AzStorageFileSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageShare'.
VERBOSE: Exporting cmdlet 'New-AzStorageShareSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageShareStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageDirectory'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageFile'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageShare'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageShareStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Set-AzStorageFileContent'.
VERBOSE: Exporting cmdlet 'Set-AzStorageShareQuota'.
VERBOSE: Exporting cmdlet 'Set-AzStorageShareStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Start-AzStorageFileCopy'.
VERBOSE: Exporting cmdlet 'Stop-AzStorageFileCopy'.
VERBOSE: Exporting cmdlet 'Disable-AzStorageDeleteRetentionPolicy'.
VERBOSE: Exporting cmdlet 'Disable-AzStorageStaticWebsite'.
VERBOSE: Exporting cmdlet 'Enable-AzStorageDeleteRetentionPolicy'.
VERBOSE: Exporting cmdlet 'Enable-AzStorageStaticWebsite'.
VERBOSE: Exporting cmdlet 'Get-AzStorageCORSRule'.
VERBOSE: Exporting cmdlet 'Get-AzStorageServiceLoggingProperty'.
VERBOSE: Exporting cmdlet 'Get-AzStorageServiceMetricsProperty'.
VERBOSE: Exporting cmdlet 'Get-AzStorageServiceProperty'.
VERBOSE: Exporting cmdlet 'New-AzStorageAccountSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageContext'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageCORSRule'.
VERBOSE: Exporting cmdlet 'Set-AzStorageCORSRule'.
VERBOSE: Exporting cmdlet 'Set-AzStorageServiceLoggingProperty'.
VERBOSE: Exporting cmdlet 'Set-AzStorageServiceMetricsProperty'.
VERBOSE: Exporting cmdlet 'Update-AzStorageServiceProperty'.
VERBOSE: Exporting cmdlet 'Set-AzStorageContainerAcl'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageBlob'.
VERBOSE: Exporting cmdlet 'Set-AzStorageBlobContent'.
VERBOSE: Exporting cmdlet 'Get-AzStorageBlob'.
VERBOSE: Exporting cmdlet 'Get-AzStorageBlobContent'.
VERBOSE: Exporting cmdlet 'Get-AzStorageBlobCopyState'.
VERBOSE: Exporting cmdlet 'Get-AzStorageContainer'.
VERBOSE: Exporting cmdlet 'Get-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'New-AzStorageBlobSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageContainer'.
VERBOSE: Exporting cmdlet 'New-AzStorageContainerSASToken'.
VERBOSE: Exporting cmdlet 'New-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageContainer'.
VERBOSE: Exporting cmdlet 'Remove-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Set-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Exporting cmdlet 'Start-AzStorageBlobCopy'.
VERBOSE: Exporting cmdlet 'Start-AzStorageBlobIncrementalCopy'.
VERBOSE: Exporting cmdlet 'Stop-AzStorageBlobCopy'.
VERBOSE: Exporting cmdlet 'Get-AzDataLakeGen2ChildItem'.
VERBOSE: Exporting cmdlet 'Get-AzDataLakeGen2ItemContent'.
VERBOSE: Exporting cmdlet 'Get-AzDataLakeGen2Item'.
VERBOSE: Exporting cmdlet 'Move-AzDataLakeGen2Item'.
VERBOSE: Exporting cmdlet 'New-AzDataLakeGen2Item'.
VERBOSE: Exporting cmdlet 'Remove-AzDataLakeGen2Item'.
VERBOSE: Exporting cmdlet 'Update-AzDataLakeGen2Item'.
VERBOSE: Exporting alias 'New-AzDataLakeGen2ItemAclObject'.
VERBOSE: Exporting alias 'Disable-AzStorageSoftDelete'.
VERBOSE: Exporting alias 'Enable-AzStorageSoftDelete'.
VERBOSE: Exporting alias 'Get-AzStorageContainerAcl'.
VERBOSE: Exporting alias 'Get-AzDatalakeGen2FileSystem'.
VERBOSE: Exporting alias 'New-AzDatalakeGen2FileSystem'.
VERBOSE: Exporting alias 'Remove-AzDatalakeGen2FileSystem'.
VERBOSE: Exporting alias 'Start-CopyAzureStorageBlob'.
VERBOSE: Exporting alias 'Stop-CopyAzureStorageBlob'.
VERBOSE: Importing cmdlet 'Add-AzRmStorageContainerLegalHold'.
VERBOSE: Importing cmdlet 'Add-AzStorageAccountManagementPolicyAction'.
VERBOSE: Importing cmdlet 'Add-AzStorageAccountNetworkRule'.
VERBOSE: Importing cmdlet 'Close-AzStorageFileHandle'.
VERBOSE: Importing cmdlet 'Disable-AzStorageBlobDeleteRetentionPolicy'.
VERBOSE: Importing cmdlet 'Disable-AzStorageDeleteRetentionPolicy'.
VERBOSE: Importing cmdlet 'Disable-AzStorageStaticWebsite'.
VERBOSE: Importing cmdlet 'Enable-AzStorageBlobDeleteRetentionPolicy'.
VERBOSE: Importing cmdlet 'Enable-AzStorageDeleteRetentionPolicy'.
VERBOSE: Importing cmdlet 'Enable-AzStorageStaticWebsite'.
VERBOSE: Importing cmdlet 'Get-AzDataLakeGen2ChildItem'.
VERBOSE: Importing cmdlet 'Get-AzDataLakeGen2Item'.
VERBOSE: Importing cmdlet 'Get-AzDataLakeGen2ItemContent'.
VERBOSE: Importing cmdlet 'Get-AzRmStorageContainer'.
VERBOSE: Importing cmdlet 'Get-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Importing cmdlet 'Get-AzRmStorageShare'.
VERBOSE: Importing cmdlet 'Get-AzStorageAccount'.
VERBOSE: Importing cmdlet 'Get-AzStorageAccountKey'.
VERBOSE: Importing cmdlet 'Get-AzStorageAccountManagementPolicy'.
VERBOSE: Importing cmdlet 'Get-AzStorageAccountNameAvailability'.
VERBOSE: Importing cmdlet 'Get-AzStorageAccountNetworkRuleSet'.
VERBOSE: Importing cmdlet 'Get-AzStorageBlob'.
VERBOSE: Importing cmdlet 'Get-AzStorageBlobContent'.
VERBOSE: Importing cmdlet 'Get-AzStorageBlobCopyState'.
VERBOSE: Importing cmdlet 'Get-AzStorageBlobServiceProperty'.
VERBOSE: Importing cmdlet 'Get-AzStorageContainer'.
VERBOSE: Importing cmdlet 'Get-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Get-AzStorageCORSRule'.
VERBOSE: Importing cmdlet 'Get-AzStorageFile'.
VERBOSE: Importing cmdlet 'Get-AzStorageFileContent'.
VERBOSE: Importing cmdlet 'Get-AzStorageFileCopyState'.
VERBOSE: Importing cmdlet 'Get-AzStorageFileHandle'.
VERBOSE: Importing cmdlet 'Get-AzStorageQueue'.
VERBOSE: Importing cmdlet 'Get-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Get-AzStorageServiceLoggingProperty'.
VERBOSE: Importing cmdlet 'Get-AzStorageServiceMetricsProperty'.
VERBOSE: Importing cmdlet 'Get-AzStorageServiceProperty'.
VERBOSE: Importing cmdlet 'Get-AzStorageShare'.
VERBOSE: Importing cmdlet 'Get-AzStorageShareStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Get-AzStorageTable'.
VERBOSE: Importing cmdlet 'Get-AzStorageTableStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Get-AzStorageUsage'.
VERBOSE: Importing cmdlet 'Lock-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Importing cmdlet 'Move-AzDataLakeGen2Item'.
VERBOSE: Importing cmdlet 'New-AzDataLakeGen2Item'.
VERBOSE: Importing cmdlet 'New-AzRmStorageContainer'.
VERBOSE: Importing cmdlet 'New-AzRmStorageShare'.
VERBOSE: Importing cmdlet 'New-AzStorageAccount'.
VERBOSE: Importing cmdlet 'New-AzStorageAccountKey'.
VERBOSE: Importing cmdlet 'New-AzStorageAccountManagementPolicyFilter'.
VERBOSE: Importing cmdlet 'New-AzStorageAccountManagementPolicyRule'.
VERBOSE: Importing cmdlet 'New-AzStorageAccountSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageBlobSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageContainer'.
VERBOSE: Importing cmdlet 'New-AzStorageContainerSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'New-AzStorageContext'.
VERBOSE: Importing cmdlet 'New-AzStorageDirectory'.
VERBOSE: Importing cmdlet 'New-AzStorageFileSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageQueue'.
VERBOSE: Importing cmdlet 'New-AzStorageQueueSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'New-AzStorageShare'.
VERBOSE: Importing cmdlet 'New-AzStorageShareSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageShareStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'New-AzStorageTable'.
VERBOSE: Importing cmdlet 'New-AzStorageTableSASToken'.
VERBOSE: Importing cmdlet 'New-AzStorageTableStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Remove-AzDataLakeGen2Item'.
VERBOSE: Importing cmdlet 'Remove-AzRmStorageContainer'.
VERBOSE: Importing cmdlet 'Remove-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Importing cmdlet 'Remove-AzRmStorageContainerLegalHold'.
VERBOSE: Importing cmdlet 'Remove-AzRmStorageShare'.
VERBOSE: Importing cmdlet 'Remove-AzStorageAccount'.
VERBOSE: Importing cmdlet 'Remove-AzStorageAccountManagementPolicy'.
VERBOSE: Importing cmdlet 'Remove-AzStorageAccountNetworkRule'.
VERBOSE: Importing cmdlet 'Remove-AzStorageBlob'.
VERBOSE: Importing cmdlet 'Remove-AzStorageContainer'.
VERBOSE: Importing cmdlet 'Remove-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Remove-AzStorageCORSRule'.
VERBOSE: Importing cmdlet 'Remove-AzStorageDirectory'.
VERBOSE: Importing cmdlet 'Remove-AzStorageFile'.
VERBOSE: Importing cmdlet 'Remove-AzStorageQueue'.
VERBOSE: Importing cmdlet 'Remove-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Remove-AzStorageShare'.
VERBOSE: Importing cmdlet 'Remove-AzStorageShareStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Remove-AzStorageTable'.
VERBOSE: Importing cmdlet 'Remove-AzStorageTableStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Revoke-AzStorageAccountUserDelegationKeys'.
VERBOSE: Importing cmdlet 'Set-AzCurrentStorageAccount'.
VERBOSE: Importing cmdlet 'Set-AzDataLakeGen2ItemAclObject'.
VERBOSE: Importing cmdlet 'Set-AzRmStorageContainerImmutabilityPolicy'.
VERBOSE: Importing cmdlet 'Set-AzStorageAccount'.
VERBOSE: Importing cmdlet 'Set-AzStorageAccountManagementPolicy'.
VERBOSE: Importing cmdlet 'Set-AzStorageBlobContent'.
VERBOSE: Importing cmdlet 'Set-AzStorageContainerAcl'.
VERBOSE: Importing cmdlet 'Set-AzStorageContainerStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Set-AzStorageCORSRule'.
VERBOSE: Importing cmdlet 'Set-AzStorageFileContent'.
VERBOSE: Importing cmdlet 'Set-AzStorageQueueStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Set-AzStorageServiceLoggingProperty'.
VERBOSE: Importing cmdlet 'Set-AzStorageServiceMetricsProperty'.
VERBOSE: Importing cmdlet 'Set-AzStorageShareQuota'.
VERBOSE: Importing cmdlet 'Set-AzStorageShareStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Set-AzStorageTableStoredAccessPolicy'.
VERBOSE: Importing cmdlet 'Start-AzStorageBlobCopy'.
VERBOSE: Importing cmdlet 'Start-AzStorageBlobIncrementalCopy'.
VERBOSE: Importing cmdlet 'Start-AzStorageFileCopy'.
VERBOSE: Importing cmdlet 'Stop-AzStorageBlobCopy'.
VERBOSE: Importing cmdlet 'Stop-AzStorageFileCopy'.
VERBOSE: Importing cmdlet 'Update-AzDataLakeGen2Item'.
VERBOSE: Importing cmdlet 'Update-AzRmStorageContainer'.
VERBOSE: Importing cmdlet 'Update-AzRmStorageShare'.
VERBOSE: Importing cmdlet 'Update-AzStorageAccountNetworkRuleSet'.
VERBOSE: Importing cmdlet 'Update-AzStorageBlobServiceProperty'.
VERBOSE: Importing cmdlet 'Update-AzStorageServiceProperty'.
VERBOSE: Importing alias 'Disable-AzStorageSoftDelete'.
VERBOSE: Importing alias 'Enable-AzStorageSoftDelete'.
VERBOSE: Importing alias 'Get-AzDatalakeGen2FileSystem'.
VERBOSE: Importing alias 'Get-AzStorageContainerAcl'.
VERBOSE: Importing alias 'New-AzDatalakeGen2FileSystem'.
VERBOSE: Importing alias 'New-AzDataLakeGen2ItemAclObject'.
VERBOSE: Importing alias 'Remove-AzDatalakeGen2FileSystem'.
VERBOSE: Importing alias 'Start-CopyAzureStorageBlob'.
VERBOSE: Importing alias 'Stop-CopyAzureStorageBlob'.
```

The Az.Accounts results in: 
```
PS C:\Users\Aaron> Import-Module -name Az.Accounts -debug
VERBOSE: Loading module from path 'C:\Users\Aaron\Documents\PowerShell\Modules\Az.Accounts\1.7.5\Az.Accounts.psd1'.

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"): a
VERBOSE: Importing cmdlet 'Add-AzEnvironment'.
VERBOSE: Importing cmdlet 'Clear-AzContext'.
VERBOSE: Importing cmdlet 'Clear-AzDefault'.
VERBOSE: Importing cmdlet 'Connect-AzAccount'.
VERBOSE: Importing cmdlet 'Disable-AzContextAutosave'.
VERBOSE: Importing cmdlet 'Disable-AzDataCollection'.
VERBOSE: Importing cmdlet 'Disable-AzureRmAlias'.
VERBOSE: Importing cmdlet 'Disconnect-AzAccount'.
VERBOSE: Importing cmdlet 'Enable-AzContextAutosave'.
VERBOSE: Importing cmdlet 'Enable-AzDataCollection'.
VERBOSE: Importing cmdlet 'Enable-AzureRmAlias'.
VERBOSE: Importing cmdlet 'Get-AzContext'.
VERBOSE: Importing cmdlet 'Get-AzContextAutosaveSetting'.
VERBOSE: Importing cmdlet 'Get-AzDefault'.
VERBOSE: Importing cmdlet 'Get-AzEnvironment'.
VERBOSE: Importing cmdlet 'Get-AzProfile'.
VERBOSE: Importing cmdlet 'Get-AzSubscription'.
VERBOSE: Importing cmdlet 'Get-AzTenant'.
VERBOSE: Importing cmdlet 'Import-AzContext'.
VERBOSE: Importing cmdlet 'Register-AzModule'.
VERBOSE: Importing cmdlet 'Remove-AzContext'.
VERBOSE: Importing cmdlet 'Remove-AzEnvironment'.
VERBOSE: Importing cmdlet 'Rename-AzContext'.
VERBOSE: Importing cmdlet 'Resolve-AzError'.
VERBOSE: Importing cmdlet 'Save-AzContext'.
VERBOSE: Importing cmdlet 'Select-AzContext'.
VERBOSE: Importing cmdlet 'Select-AzProfile'.
VERBOSE: Importing cmdlet 'Send-Feedback'.
VERBOSE: Importing cmdlet 'Set-AzContext'.
VERBOSE: Importing cmdlet 'Set-AzDefault'.
VERBOSE: Importing cmdlet 'Set-AzEnvironment'.
VERBOSE: Importing cmdlet 'Uninstall-AzureRm'.
VERBOSE: Importing alias 'Add-AzAccount'.
VERBOSE: Importing alias 'Get-AzDomain'.
VERBOSE: Importing alias 'Login-AzAccount'.
VERBOSE: Importing alias 'Logout-AzAccount'.
VERBOSE: Importing alias 'Remove-AzAccount'.
VERBOSE: Importing alias 'Resolve-Error'.
VERBOSE: Importing alias 'Save-AzProfile'.
VERBOSE: Importing alias 'Select-AzSubscription'.
```

**Comment by dingmeng-xue** [comment]
@ADataGMan, this issue is usually due to one old version assembly is loaded before import Az.Accounts. As the result, it reports the issue. It is behavior on dotnet core. We only can ensure Az modules have no that issue but the situation is complex if there are many other modules.
Do you still have any other issue?

**Comment by ADataGMan** [comment]
At this point my issues seem to be resolved, but beyond doing the same
delete everything / reboot / reinstall everything / reboot cycle I'm not
sure why it doesn't seem to be an issue now. Maybe the installation of the
modules as an admin, and trying to use them in another context meant they
couldn't be picked up?

This can be closed, I'll submit a new issue if it comes back.

**Comment by dingmeng-xue** [comment]
Hi ADataGMan, it's a real issue to you and customer. Couple issues are pointing to this limitation. We had complained this issue couple time to PowerShell team and PowerShell had been investigating it for a while. We are waiting their solution.
Thanks for reporting. Please contact us if you meet the issue.
